integration/auth/user_mgt_test.go

```diff
@@ -91,6 +92,258 @@ func TestGetNonExistingUser(t *testing.T) {
 	}
 }
 
+func TestGetUsers(t *testing.T) {
+	testUser1 := (&auth.UserToCreate{}).
+		UID("uid1").
+		Email("user1@example.com").
+		PhoneNumber("+15555550001")
+	testUser2 := (&auth.UserToCreate{}).
+		UID("uid2").
+		Email("user2@example.com").
+		PhoneNumber("+15555550002")
+	testUser3 := (&auth.UserToCreate{}).
+		UID("uid3").
+		Email("user3@example.com").
+		PhoneNumber("+15555550003")
```

Comment:
Use `newUserWithParams()` to create the first 3 users. Then you can get rid of the `createUser()` helper function.